__tests__/commands/info.js

```diff
@@ -47,7 +48,7 @@ const expectedKeys = [
 ];
 
 test.concurrent('without arguments and in directory containing a valid package file', (): Promise<void> => {
-  return runInfo(['.'], {}, 'local',
+  return runInfo([], {}, 'local',
```

Comment:
NOTE: this looks like I'm removing the `['.']` case, but there were actually 2 of the exact same test cases in this file.  All I'm doing here is making this test case match the description (and hereby improving the coverage report).